packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-expandable-searchbox {
+  width: var(--_iui-input-min-height);
+  transition: width var(--iui-duration-1) ease-in-out;
+  isolation: isolate;
```

Comment:
this should go in `input-flex-container.scss` because there is no z-index in this file